packages/react/src/components/Tag/DismissibleTag.tsx

```diff
@@ -111,7 +111,7 @@ const DismissibleTag = <T extends React.ElementType>({
   };
 
   let normalizedSlug;
```

Comment:
quick question before I fully review, should we renamethese also? (slug, normalizedSlug, slugRef etc)?

Reply:
I wasn’t planning on changing the prop names for each component, so I didn’t touch the internal variable names either 🤷‍♂️

Reply:
Makes sense to me 👍